kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -166,7 +166,7 @@ define([
 
             // Add ACL before going to the staging area
             // If it fails, it'll just do so silently.
-            var $globusLink = this.$elem.find('.globus_link');
+            var $globusLink = this.$elem.find('a.globus_linked');
```

Comment:
I'm not sure if I'm looking at the wrong file, but in `kbase-extension/static/kbase/templates/data_staging/ftp_file_header.html` below, it's a button with the class `globus_linked` and the `a` element has the class `globus_link`. What am I missing?

Reply:
Ah good catch: I don't think we need the globus_link class on the <a> element in ftp_file_header - just an artifact at this point. 

For context - the original class name was "globus_link" but I found it somewhat confusing since it referred to two separate links (one where the user was connected/linked to globus and one where not), hence the rename.

Reply:
fixed!